examples/mobile/UIComponents/components/view-controls/main-tab/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
```

Comment:
These pages shouldn't have titles attribute?

Reply:
Title is not required.